content/4_Gold/Paths_Grids.problems.json

```diff
@@ -79,6 +79,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "cses-3359",
+      "name": "Minimal Grid Path",
+      "url": "https://cses.fi/problemset/task/3359/",
+      "source": "CSES",
+      "difficulty": "Easy",
```

Comment:
hm, why is this easy

Reply:
Can you share your solution?

My original idea was rather overcomplicated, but I still think this should at least a difficulty higher than the problems before it and the Gold problem after it. All of the other problems rated easy use pretty much the same idea, but this one doesn't.

Reply:
https://cses.fi/paste/6dd211c3a7f931cdc584fa/